### PR TITLE
adds tenant existence check endpoint

### PR DESCRIPTION
**Pull request by esotrope**

### CODE REVIEW

File: admin-service/src/main/java/org/opentestsystem/rdw/admin/multitenant/service/impl/DefaultTenantConfigurationViewService.java [hunk 1]
```diff
@@ -221,9 +222,27 @@ private String getLanguageCode(final List<String> uiLanguages) {
...
+        return allInProcessTenantConfigurations()
+                .entrySet()
+                .stream()
+                .filter(entry -> entry.getValue().getTenant().isSandbox() == (tenantType == TenantType.SANDBOX))
+                .collect(Collectors.toMap(Map.Entry::getKey, Map.Entry::getValue));
+    }
+
+    @VisibleForTesting
+    Map<String, TenantConfiguration> allInProcessTenantConfigurations() {
+        final Map<String, Tenant> currentSandboxTenants = activeTenants();
```
Comment (esotrope) [inline admin-service/src/main/java/org/opentestsystem/rdw/admin/multitenant/service/impl/DefaultTenantConfigurationViewService.java]: is this supposed to be active tenants or in progress?

Comment (esotrope) [reply admin-service/src/main/java/org/opentestsystem/rdw/admin/multitenant/service/impl/DefaultTenantConfigurationViewService.java]: @ftdholbrook

Comment (ftdholbrook) [reply admin-service/src/main/java/org/opentestsystem/rdw/admin/multitenant/service/impl/DefaultTenantConfigurationViewService.java]: allInProcess

Comment (esotrope) [reply admin-service/src/main/java/org/opentestsystem/rdw/admin/multitenant/service/impl/DefaultTenantConfigurationViewService.java]: 👍 k i'll update

Comment (ftdholbrook) [reply admin-service/src/main/java/org/opentestsystem/rdw/admin/multitenant/service/impl/DefaultTenantConfigurationViewService.java]: What needs updating, I don't understand.

Comment (ftdholbrook) [reply admin-service/src/main/java/org/opentestsystem/rdw/admin/multitenant/service/impl/DefaultTenantConfigurationViewService.java]: ahh neve mind I see sorry too many context switches

**Pull request merged**